cloud/k8s.go

```diff
@@ -180,7 +180,17 @@ func (cloud *K8SCloud) CanProvision(quota Quota) (bool, error) {
 
 // Provision returns a worker if the cloud can provison
 func (cloud *K8SCloud) Provision(id string, wopts WorkerOptions) (Worker, error) {
-	can, err := cloud.CanProvision(wopts.Quota)
+	var cp *K8SCloud
```

Comment:
What does cp mean?

Reply:
`cp` is short for `cloud provider`. It is just a local variable, so the the abbreviation.